core/network/src/protocol.rs

```diff
@@ -38,9 +40,13 @@ use crate::chain::Client;
 use client::light::fetcher::ChangesProof;
 use crate::{error, util::LruHashSet};
 
+/// Interval at which we send status updates on the SyncProvider status stream.
+pub const STATUS_INTERVAL_MS: u64 = 5000;
+
 const REQUEST_TIMEOUT_SEC: u64 = 40;
 const TICK_TIMEOUT: time::Duration = time::Duration::from_millis(1000);
-const PROPAGATE_TIMEOUT: time::Duration = time::Duration::from_millis(5000);
+const PROPAGATE_TIMEOUT: time::Duration = time::Duration::from_millis(STATUS_INTERVAL_MS + 100);
```

Comment:
I guess it's changed to avoid two timers firing at the same time? I'd rather make it more frequent though.

Reply:
Yes I thought if they are both set for 5000 ms that is surely going to make the status interval less reliable. 

Do you mean that the propagate should be more frequent? It relates to propagating extrinsincs.

Reply:
Yeah, I'd say that propagation is a bit more important, not sure why it was as high as 5s in the first place though. I'd use timeouts that conflict as rarely as possible - currently it seems that every fifth tick will also conflict with status. Anyways I think we can safely change propagate to something like `4100ms`